src/main/java/com/sap/cloudfoundry/client/facade/rest/CloudControllerRestClientImpl.java

```diff
@@ -2104,13 +2097,13 @@ private Flux<DomainResource> getDomainResources() {
     private Flux<DomainResource> getSharedDomainResources() {
         return getDomainResources().filter(domain -> domain.getRelationships()
                                                            .getOrganization()
-                                                           .getData() != null);
+                                                           .getData() == null);
```

Comment:
Are these intentionally chenged

Reply:
Yes. The API docs have erroneous descriptions of "shared" and "private" domains. 
I changed the logic based on the CF CLI.